src/Engine/App/AppBase.cpp

```
@@ -54,7 +54,7 @@ using namespace ::OSRE::Animation;
 using namespace ::OSRE::Properties;
 using namespace ::OSRE::IO;
 
-static constexpr c8 Tag[] = "AppBase";
+DECL_OSRE_LOG_MODULE(AppBase)
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If DECL_OSRE_LOG_MODULE is a macro then please configure it.

(unknownMacro)